components/showcard.tsx

```diff
@@ -0,0 +1,106 @@
+import { format } from 'date-fns';
+import fi from 'date-fns/locale/fi';
+import Image from 'next/image';
+import { useRef, useState } from 'react';
+
+import { ShowsCollectionItem } from 'pages/arkisto/[showlistId]';
+import { contentfulImageLoader } from 'contentful/contentfulImageLoader';
+import placeholderImage from '../public/kuva_puuttuu_v2.jpeg';
+
+interface ShowCard {
+  show: ShowsCollectionItem;
+  index: number;
+}
+
+export const ShowCard = ({ show, index }: ShowCard) => {
+  const [isExpanded, setIsExpanded] = useState<boolean>(false);
+  const ref = useRef(null);
+
+  const { picture } = show;
+  const url = picture?.url || placeholderImage;
+  const loader = picture?.url ? contentfulImageLoader : undefined;
+
+  return (
+    <div
+      className={`mx-auto flex w-full rounded ${
+        isExpanded
+          ? 'h-auto rounded-l-none rounded-t lg:rounded-l lg:rounded-t-none'
+          : 'h-52'
+      }`}
+    >
```

Comment:
Mä litistäisin näitä kuvia myös mobiilissa jos kortti on kiinni. Mahtuu vähän enempi kamaa ruudulle.